custom-options-plus.php

```diff
@@ -44,10 +44,21 @@
 
 global $wpdb, $COP_TABLE;
 define( 'COP_TABLE', $wpdb->prefix . 'custom_options_plus' );
+define('COP_SECURITY_STRING', date('h'));
```

Comment:
I see your point. But it's unnecessary.

Please, read this article https://codex.wordpress.org/WordPress_Nonces

In Portuguese you can read that one https://blog.apiki.com/2015/09/21/wordpress-nonces-e-o-que-voce-precisa-saber-sobre-isso/